cvat/apps/iam/permissions.py

```diff
@@ -95,10 +95,12 @@ def get_iam_context(request, obj):
         package, attr = builder_func_path.rsplit('.', 1)
         builder_func = getattr(import_module(package), attr)
         iam_context.update(builder_func(request, organization, membership))
-
     # FIXME: The primary app should know nothing about is_crowdsourcing plugin.
     if organization and not request.user.is_superuser and membership is None and not iam_context.get('is_crowdsourcing', False):
-        raise PermissionDenied({'message': 'You should be an active member in the organization'})
+        # Allow inactive members view, accept, reject their invitations
+        view = request.parser_context.get('view')
+        if not (view.basename == 'invitation' and view.action in ['retrieve', 'accept', 'reject']):
```

Comment:
Why do we need the logic here?

Reply:
PermissionDenied error is raised here if organization is not None and membership is None. That is the case for Invitation objects(they belong to some org and user that tries to accept invitation is not an active member). So we handle this case and do not throw an error.

Reply:
From the first look it is wrong place for the logic.